contracts/package.json

```diff
@@ -18,7 +18,7 @@
   "author": "James Zaki",
   "license": "MIT",
   "dependencies": {
-    "@openzeppelin/contracts": "^4.7.3"
+    "@openzeppelin/contracts": "^4.8.0"
```

Comment:
Is it okay if I bump this up?

Reply:
I don't see any changes that should impact us. @jzaki can give final say. 

Generally, in earlier stages of a project it is fine to bump this at will. Later on, it can be less desirable as:
- Many OpenZepplin updates have breaking changes, even in non-major version changes. This version jump [has 3](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.8.0)
- Our contracts will be audited at a specific commit/tag with a specific OpenZepplin version used. Bumping this *could* require a new audit pass/tests.
- This will likely change our contract byte code, requiring new contract deploys and client lib updates.